mne/label.py

```diff
@@ -2666,3 +2669,49 @@ def select_sources(subject, label, location='center', extent=0.,
                           hemi=new_label.hemi, name=name, subject=subject)
 
     return new_label
+
+
+def find_label_in_annot(pos, subject='fsaverage', annot='aparc.a2005s+aseg',
+                        subjects_dir=None):
+    """
+    Find name in atlas for given MRI coordinates.
+
+    Parameters
+    ----------
+    pos : ndarray, shape (3,)
+        Vector of x,y,z coordinates in MRI space.
+    subject : str
+        MRI subject name.
+    annot : str
+        MRI parcellation file name. Do not include the ``.mgz`` suffix.
```

Comment:
Eventually we probably want to add support for nearest-surface-label functionality using the `lh.aparc.annot` (and similar files) and the cortical ribbon, so for now let's make it clear that it's going to use a volumetric atlas only at this point:
```suggestion
        MRI volumetric atlas file name.
        Do not include the ``.mgz`` suffix.
```